### PR TITLE
User/crloewen/new in 1903

### PR DESCRIPTION
**Pull request by mscraigloewen**
Added an extra section to the documentation to reflect the changes made to wsl.exe as detailed in this blog post:
https://blogs.msdn.microsoft.com/commandline/2019/02/15/whats-new-for-wsl-in-windows-10-version-1903/

### CODE REVIEW

File: WSL/wsl-config.md [hunk 1]
```diff
@@ -117,6 +117,49 @@ PS C:\Users\sarah>
 
 ## Managing multiple Linux Distributions
 
+### Windows 10 Version 1903 and later
+
+You can use `wsl.exe` to manage your multiple distributions in the Windows Subsystem for Linux (WSL), including listing available distributions, setting a default distribution, and uninstalling distributions.
```
Comment (benhillis) [inline WSL/wsl-config.md]: remove multiple

File: WSL/wsl-config.md [hunk 2]
```diff
@@ -117,6 +117,49 @@ PS C:\Users\sarah>
...
+
+Each Linux distribution independently manages its own configurations. To see distribution-specific commands, run `[distro.exe] /?`.  For example `ubuntu /?`.
+
+#### List distributions
+
+`wsl -l` , `wsl --list`  
+Lists available Linux distributions available to WSL.  If a distribution is listed, it's installed and ready to use.
+
+`wsl --list --all`   
+Lists all distributions, including ones that aren't currently usable.  They may be in the process of installing, uninstalling, or are in a broken state.
```
Comment (benhillis) [inline WSL/wsl-config.md]: Missing wsl --list --running

File: WSL/wsl-config.md [hunk 3]
```diff
@@ -117,6 +117,49 @@ PS C:\Users\sarah>
...
+
+> **Caution:** Once unregistered, all data, settings, and software associated with that distribution will be permanently lost.  Reinstalling from the store will install a clean copy of the distribution.
+
+`wsl --unregister <DistributionName>`  
+Unregisters the distribution from WSL so it can be reinstalled or cleaned up.
+
+For example:
+`wsl -unregister Ubuntu` would remove Ubuntu from the distributions available in WSL.  When I run `wsl --list` it will not be listed.
+
+To reinstall, find the distribution in the Windows Store and select "Launch".
```
Comment (benhillis) [inline WSL/wsl-config.md]: Missing --user and --distribution arguments.

**Comment by mscraigloewen** [comment]
Changes added! Merging the pull request now.

**Pull request merged**